### PR TITLE
LPS-130985 Add MetricsCalculatedInfo in performance tab

### PR DESCRIPTION
**Pull request by ramonmsp**
Story: [LPS-94760](https://issues.liferay.com/browse/LPS-94760)

### CODE REVIEW

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/App.es.js [hunk 1]
```diff
@@ -139,6 +139,7 @@ describe('The App component should', () => {
 	});
 
 	test('Render the process metrics page on performance tab and back to dashboard', () => {
+		const metricsCalculated = findByText('Metrics calculated');
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/App.es.js:142]: Hey @ramonmsp, Is the `findByText` case insensitive? Thx

**Comment by naraandrade** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Review by naraandrade — APPROVED** [review]

**Review by jotascript — APPROVED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/App.es.js (hunk 1 above)
Comment (ramonmsp) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/App.es.js:142]: It was wrong, sorry :(

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**